pkg/rtc/subscribedtrack.go

```diff
@@ -36,10 +37,13 @@ type SubscribedTrack struct {
 }
 
 func NewSubscribedTrack(params SubscribedTrackParams) *SubscribedTrack {
-	return &SubscribedTrack{
+	s := &SubscribedTrack{
 		params:    params,
 		debouncer: debounce.New(subscriptionDebounceInterval),
 	}
+
+	s.params.DownTrack.SetMaxSpatialLayer(SpatialLayerForQuality(livekit.VideoQuality_HIGH))
```

Comment:
This can be removed when there is protocol support. Idea would be to call this based on protocol setting.